assets/js/index.js

```diff
@@ -7,6 +8,7 @@ require('bootstrap');
 require('select2');
 
 $(() => {
+	base();
```

Comment:
Regarding the arrow function. Afaik we are not transpiling the code. Thus it won't work in IE.